app/views/shared/_transaction_variant.html.erb

```diff
@@ -9,53 +9,82 @@
   </div>
   <div id="<%= f.object.slug || 'untitled-part' %>" class="js-part-toggle-target panel-collapse collapse in" aria-expanded="true">
     <div class="panel-body">
-        <%= f.inputs do %>
-          <%= f.input :title,
-                      :input_html => { :class => 'title', :disabled => @resource.locked_for_edits? },
-                      :hint => false,
-                      :required => true %>
```

Comment:
Does this mean that there is an asterisk next to the label? i.e. adding `<abbr title="required">*</abbr>`